system/xmllib/libxml_xpathprocessor.cpp

```diff
@@ -1628,10 +1631,11 @@ class CEsdlScriptContext : public CInterfaceOf<IEsdlScriptContext>
     xmlDocPtr doc = nullptr;
     xmlNodePtr root = nullptr;
     xmlXPathContextPtr xpathCtx = nullptr;
+    IInterface *functionRegister = nullptr;
```

Comment:
Should this be IEsdlFunctionRegister?

Reply:
This was made opaque to ovoid having to avoid dependencies and have to move declarations outside of where they are used.  But I think Tim's upcoming changes may move this and some other similar items into a new derived class.  I'd prefer to wait and see what happens with that before major refactoring here.